libva will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libva it will fail in the following way:

```
2024/11/12 23:12:32 INFO running step "test/pkgconf"
2024/11/12 23:12:32 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:32 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:32 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:32 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:32 INFO 1.22.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 INFO -lva-drm -lva uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 INFO uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + basename /home/build/melange-out/libva-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + dev_pkg=libva-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + apk info -L libva-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q ^Name: usr/lib/pkgconfig/libva-drm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + basename usr/lib/pkgconfig/libva-drm.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + lib_name=libva-drm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + echo usr/lib/pkgconfig/libva-drm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q '^usr/lib/pkgconfig/libva-drm.pc$\|^usr/share/pkgconfig/libva-drm.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + pkgconf --exists libva-drm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q ^Version: usr/lib/pkgconfig/libva-drm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + pkgconf --modversion libva-drm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q ^Libs: usr/lib/pkgconfig/libva-drm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + pkgconf --libs libva-drm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q ^Cflags: usr/lib/pkgconfig/libva-drm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + pkgconf --cflags libva-drm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q ^Name: usr/lib/pkgconfig/libva-wayland.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + basename usr/lib/pkgconfig/libva-wayland.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + lib_name=libva-wayland uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + echo usr/lib/pkgconfig/libva-wayland.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + grep -q '^usr/lib/pkgconfig/libva-wayland.pc$\|^usr/share/pkgconfig/libva-wayland.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + pkgconf --exists libva-wayland uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN + pkgconf --print-errors --exists libva-wayland uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN Package wayland-client was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN Perhaps you should add the directory containing `wayland-client.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 WARN Package 'wayland-client', required by 'libva-wayland', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:32 INFO pod d9f608f5cedc6808e82ada8289a7e43a5728b357138abd23d27f152334bf2db1 terminated
2024/11/12 23:12:32 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:32 INFO   workspace dir: /home/user/tmp/melange-workspace-3316891387
2024/11/12 23:12:32 INFO   guest dir: /home/user/tmp/melange-guest-1960246349
2024/11/12 23:12:32 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libva] Error 1
```

--- a/libva.yaml
+++ b/libva.yaml
@@ -2,7 +2,7 @@
 package:
   name: libva
   version: 2.22.0
-  epoch: 0
+  epoch: 1
   description: Video Acceleration (VA) API for Linux
   copyright:
     - license: MIT
@@ -54,3 +54,7 @@ update:
   enabled: true
   github:
     identifier: intel/libva
+
+test:
+  pipeline:
+    - uses: test/pkgconf

